<launch>

  <arg name="UAV_NAME" default="$(env UAV_NAME)"/>

  <group ns="$(arg UAV_NAME)">
    
    <node name="RandomFlier" pkg="nodelet" type="nodelet" args="standalone mrs_testing/RandomFlier" output="screen" >
      
      <rosparam file="$(find mrs_testing)/config/random_flier.yaml" />

      <!-- Subscribers -->
      <remap from="~control_cmd_in" to="control_manager/cmd_odom" />

      <!-- Publishers -->
      <remap from="~goto_out" to="control_manager/goto" />
    
      <!-- Services -->
      <remap from="~activate_in" to="~activate" />

    </node>
  
  </group>
</launch>
